### ISSUE TITLE
Broken keyboard navigation

### ISSUE THREAD

**Issue by juni-vogt** [laconalabs/LaconaApp#76]
Some keyboard navigation doesn't work right now
cmd+left / cmd+right – should jump to beginning/end of line
alt+left / alt+right – should jump to beginning/end of word

**Comment by brandonhorst** [comment]
Fixed with [0.6](https://github.com/laconalabs/LaconaApp/blob/master/CHANGELOG.md). Thanks for reporting!
